rocky/reports/templates/report_overview/report_history_table.html

```diff
@@ -1,151 +1,231 @@
 {% load i18n %}
+{% load static %}
 {% load ooi_extra %}
 {% load report_extra %}
+{% load component_tags %}
+{% load compress %}
 
-{% if reports %}
-    <p class="de-emphasized">
-        {% blocktranslate with length=reports|length total=total_oois %}Showing {{ length }} of {{ total }} reports{% endblocktranslate %}
-    </p>
-    <div class="horizontal-scroll">
-        <table>
-            <caption class="visually-hidden">{% translate "Reports:" %}</caption>
-            <thead>
-                <tr>
-                    <th scope="col">{% translate "Name" %}</th>
-                    <th scope="col">{% translate "Report type" %}</th>
-                    <th scope="col">{% translate "Input Objects" %}</th>
-                    <th scope="col">{% translate "Reference date" %}</th>
-                    <th scope="col">{% translate "Creation date" %}</th>
-                    <th scope="col"></th>
-                </tr>
-            </thead>
-            <tbody>
-                {% for report in reports %}
-                    <tr>
-                        <td>
-                            <a href="{% url "view_report" organization.code %}?report_id={{ report.parent_report.reference }}&observed_at={{ report.parent_report.observed_at|date:"Y-m-d H:i:s:u" }}"
-                               title="{% translate "Shows parent report details" %}">{{ report.parent_report.name }}</a>
-                        </td>
-                        <td>
-                            <ul class="tags horizontal-view">
-                                {% if report.parent_report.report_type == "concatenated-report" or report.parent_report.report_type is None %}
-                                    {% for report_type, total_objects in report.report_type_summary.items %}
-                                        {% if forloop.counter0 < 2 %}
-                                            <li class="label tags-color-{{ report_type|get_report_type_label_style }}">{{ report_type|get_report_type_name }}</li>
-                                        {% endif %}
-                                        {% if forloop.counter0 == 2 %}
-                                            <li class="label tags-color-grey-2">+ {{ report.report_type_summary|length|add:"-2" }}</li>
+<div class="horizontal-view toolbar">
+    <div class="dropdown">
+        {% comment %} <p class="visually-hidden" id="language-selector-description">{% translate "Select your language" %}</p> {% endcomment %}
+        <button aria-label="{{ aria_label_current_language }}"
+                type="button"
+                aria-haspopup="listbox"
+                aria-current="true"
```

Comment:
The use of `aria-current` here does not seem correct. In addition, I don't think a listbox makes sense for this dropdown. Since the list items are links, and used to navigate to the corresponding form (by opening a modal dialog), it might make more sense to apply the [Menu Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/) instead.